ui/src/app/app.component.ts

```diff
@@ -233,9 +233,9 @@ export class AppComponent implements AfterViewInit {
   }
 
   buildDownloadLink(download: Download) {
-    let baseDir = 'download/';
+    let baseDir = this.downloads.configuration["PUBLIC_HOST_URL"] ?? 'download/';
```

Comment:
These variables already have defaults, so it seems we don't need a fallback to a default here.